apps/console/src/features/authorization/hooks/use-authorization.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import getLegacyAuthzRuntime from "../utils/get-legacy-authz-runtime";
+
+/**
+ * Interface to store use authorization hook details.
+ */
+export interface UseAuthorizationInterface {
```

Comment:
Shall we move this to models?

Reply:
Moved to this file based on the above comment, https://github.com/wso2/identity-apps/pull/4368#discussion_r1373661010